packages/vinxi/lib/manifest/dev-server-manifest.js

```diff
@@ -51,7 +51,7 @@ export function createDevManifest(app) {
 								} else {
 									return {
 										output: {
-											path: absolutePath,
+											path: join(router.base, "@fs", absolutePath),
```

Comment:
this change will not in some modes: this change will not work, on the server we don't want to use @fs links. those are for the client. this is being used with the server actions setup. I should add a test for that

Reply:
@nksaraf What about LN 175? Should that also be reverted in that case? I did this in two places.

https://github.com/nksaraf/vinxi/blob/2fa85e20506f63adfd50d367aed1e4ed7e521e5a/packages/vinxi/lib/manifest/dev-server-manifest.js#L175